### PR TITLE
Unify `#@linksto` usage

### PR DESCRIPTION
**Pull request by m7pr**
This does not change any behavior of anything.
This is just consistency management with `#@linksto` tag that we have for `lockEnvironment(.raw_data)`

https://github.com/search?q=repo%3Ainsightsengineering%2Fteal+%23+%40linksto&type=code

image

### CODE REVIEW

**Review by averissimo — APPROVED** [review]

File: R/module_init_data.R [hunk 1]
```diff
@@ -128,7 +128,7 @@ srv_init_data <- function(id, data) {
     function(dataname, datasets) {
       hash <- rlang::hash(data[[dataname]])
       sprintf(
-        "stopifnot(%s == %s) # @linksto %s",
+        "stopifnot(%s == %s) #@linksto %s",
```
Comment (averissimo) [inline R/module_init_data.R:131]: This is always inside strings, however the styler ruleset that we have in insightengineering enforces a space after the `#`

Would it be better to change the other instance?

Comment (m7pr) [reply R/module_init_data.R:131]: No strong opinion as it is inside a string. Without the space there is less characters :D

Comment (m7pr) [reply R/module_init_data.R]: I'm having second thoughts and I would actually write # comment (hasthag space somment) when I would type a comment, so let's go with your approach

Comment (averissimo) [reply R/module_init_data.R]: Sounds good either way.

**Comment by m7pr** [comment]
Hey @averissimo I unified the comments for the `@linksto` tag the other way around!

**Review by averissimo — APPROVED** [review]
Awesome! :+1:

**Pull request merged**